zipline/algorithm.py

```diff
@@ -129,7 +129,7 @@ def __init__(self, *args, **kwargs):
             script : str
                 Algoscript that contains initialize and
                 handle_data function definition.
-            data_frequency : str (daily, hourly or minutely)
```

Comment:
@warren-oneill the numpy/scipy ecosystem convention for arguments with a fixed set of possible values is to write:

```
argument_name : type, {'value1', 'value2'}
```

See for example `np.sort`:
```
Parameters
----------
kind : {'quicksort', 'mergesort', 'heapsort'}, optional
    Sorting algorithm. Default is 'quicksort'.
```